velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "SubstraitToVeloxExpr.h"
+#include "velox/connectors/hive/HiveConnector.h"
+
+namespace facebook::velox::substrait {
+
+/// This class is used to convert the Substrait plan into Velox plan.
+class SubstraitVeloxPlanConverter {
+ public:
+  /// Used to convert Substrait AggregateRel into Velox PlanNode.
+  std::shared_ptr<const core::PlanNode> toVeloxPlan(
+      const ::substrait::AggregateRel& sAgg);
+
+  /// Used to convert Substrait ProjectRel into Velox PlanNode.
+  std::shared_ptr<const core::PlanNode> toVeloxPlan(
+      const ::substrait::ProjectRel& sProject);
+
+  /// Used to convert Substrait FilterRel into Velox PlanNode.
+  std::shared_ptr<const core::PlanNode> toVeloxPlan(
+      const ::substrait::FilterRel& sFilter);
+
+  /// Used to convert Substrait ReadRel into Velox PlanNode.
```

Comment:
can we explain what the parameters below mean?

Reply:
Added explanation for them.